dependencies.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import sys
+import subprocess
+import json
+
+# Set default logging level to INFO
+logging.basicConfig(level="INFO")
+
+# Check if current python version is 3.8
+if sys.version_info < (3, 8):
+    logging.fatal('McPy needs Python version 3.8.0 or higher to run! Current version is %s.%s.%s' % (
+        sys.version_info.major, sys.version_info.minor, sys.version_info.micro))
+    sys.exit(-4)
+
+# If the OS is Linux, the module "distro" will import
+try:
+    import distro
+except ModuleNotFoundError: # Windows and MacOS will not find this module, because they don't have distributions
+    pass
+
+# Make sure pip is installed
+logging.info("Making sure pip is installed...")
+try: # Debian and Ubuntu require pip to be installed differently because they disable ensurepip
+    linuxDistro = distro.linux_distribution()
```

Comment:
I am looking for specific Linux distributions, since the ensurepip error only occurs on Debian-based distros, where the os.name and platform.system() functions get full OS names with release numbers/architecture etc. The distro.linux_distribution() function returns a much easier to parse OS name (i.e. "Ubuntu", "Debian", or "Manjaro").